src/models/BotSubscription.ts

```
@@ -6,6 +6,8 @@ import mongoose, { Document } from "mongoose";
 export interface IBotSubscription extends Document {
   userId: mongoose.Types.ObjectId;
   botId: mongoose.Types.ObjectId;
+  botPackageId:mongoose.Types.ObjectId;
+  lotSize: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)